module/appsv2/allocate-chips-marshal.mjs

```diff
@@ -0,0 +1,103 @@
+/* eslint-disable no-underscore-dangle */
+
+const { ApplicationV2, HandlebarsApplicationMixin } = foundry.applications.api;
+
+export class AllocateChipsMarshal extends HandlebarsApplicationMixin(
+  ApplicationV2
+) {
+  /** @inheritdoc */
+
+  static DEFAULT_OPTIONS = {
+    form: {
+      handler: AllocateChipsMarshal.formHandler,
+      submitOnChange: false,
+      closeOnSubmit: true,
+    },
+    tag: 'form',
+    scrollY: ['.directory-list'],
+    window: {
+      icon: '',
+      title: '',
+    },
+  };
+
+  static PARTS = {
+    form: {
+      template:
+        'systems/deadlands-classic/templates/chip/allocator-marshal.html',
+    },
+  };
+
+  /* -------------------------------------------- */
+  /*  Methods                                     */
+  /* -------------------------------------------- */
+
+  /** @inheritdoc */
+  async _prepareContext(options = {}) {
+    let context = await super._prepareContext(options);
+
+    // Get the actors who can hold chips
+    const chipBearers = game.actors.filter((a) => a.system.hasChips === true);
+
+    const posse = [];
+
+    // eslint-disable-next-line no-restricted-syntax
```

Comment:
hmm we need to adjust our eslint rules if it's complaining about stuff like that

Reply:
I looked up the thing it's complaining about and it was suggested that it was maybe a good idea more than 10 years ago, but javascript has moved on. Happy to remove this rule, I just disable it when it complains anyway.